packages/remote-logs/src/logger.ts

```diff
@@ -129,32 +129,34 @@ export class RemoteLogger extends BifoldLogger {
     }, expirationInMinutes * 60000)
   }
 
-  public test(message: string, data?: object | undefined): void {
-    this._log?.test({ message, data })
+  public test(...msgs: unknown[]): void {
+    this._log?.test(...msgs)
   }
 
-  public trace(message: string, data?: object | undefined): void {
-    this._log?.trace({ message, data })
+  public trace(...msgs: unknown[]): void {
+    this._log?.trace(...msgs)
   }
 
-  public debug(message: string, data?: object | undefined): void {
-    this._log?.debug({ message, data })
+  public debug(...msgs: unknown[]): void {
+    this._log?.debug(...msgs)
   }
 
-  public info(message: string, data?: object | undefined): void {
-    this._log?.info({ message, data })
+  public info(...msgs: unknown[]): void {
+    this._log?.info(...msgs)
   }
 
-  public warn(message: string, data?: object | undefined): void {
-    this._log?.warn({ message, data })
+  public warn(...msgs: unknown[]): void {
+    this._log?.warn(...msgs)
   }
 
-  public error(message: string, data?: object | undefined): void {
-    this._log?.error({ message, data })
+  public error(...msgs: unknown[]): void {
+    const stack = new Error().stack ?? ''
```

Comment:
this logs where the logging is being called, not where the error got raised

Reply:
@MKGpp you could try checking for the first instance of an `Error` object (BifoldError and CredoError both extend the base Error object) in `msgs` and log that error's stack if it exists. That might be the easiest way to add the proper stacktrace without having to make too many changes right away. Then we can go through and make sure when we call `logger.error(...)`, we're passing an Error object of some kind.

Reply:
We do have a bit of bad pattern of creating `new BifoldError`s and only supplying the message property from the original error, losing the more accurate stacktrace (eg. L114 of RootStack.tsx) but the above would still be an improvement in the interim. Not sure if we want to address fixing that pattern now while we're working on this - the pattern is pretty widespread